server/src/main/java/io/spine/server/event/ERepository.java

```diff
@@ -92,6 +94,11 @@ void store(Event event) {
         store(entity);
     }
 
+    void store(Iterable<Event> events) {
+        final Iterable<EEntity> entities = Iterables.transform(events, EventToEEntity.instance());
+        store(newLinkedList(entities));
```

Comment:
As long as you don't ever append new items, it should probably be `newArrayList(..)`.